### DISCUSSION TITLE
Do we have a publicly viewable agenda for the weekly community call?

### DISCUSSION THREAD

**Discussion by jordandsullivan** [unitaryfund/mitiq#2294] [Q&A] [answered]
I was thinking it would be useful to have a place on Github where we could link discussions and issues we would like feedback on from the community

**Comment by purva-thakre** [comment]
We used to have it linked in `Readme`. I don't remember when that changed.

**Reply by natestemen** [reply]
Yeah, the [agenda](https://docs.google.com/document/d/1lZfct4AOCS7fdyWkudcGyER0n0nsCxSFKSicUEeJgtA) itself is publicly accessible, and generally it's good practice as the milestone manager to share the link at the beginning of the community calls, but not everyone remembers 🤷🏻. I don't think **that** recommendation, or any recommendation for running the meetings, is documented.

Having a section in the readme regarding our community calls with a link to this agenda isn't a bad idea. Only potential risky thing is that having a link (as visible as in a readme) to a publicly editable google doc does open us up to potential spam. IDK how big that risk is, though. I say we cross that bridge when we get there.

**Reply by purva-thakre** [reply]
> Having a section in the readme regarding our community calls with a link to this agenda isn't a bad idea.

What if we only link the non-editable version in the readme? During the community call, the milestone manager could share the editable version.

**Reply by natestemen** [reply, marked as answer]
I've actually convinced myself that having the link public is okay, at least to start. I want people to be able to add it it ahead of the meeting, even if they haven't attended before.

@jordandsullivan could you make a PR to add this to the README?

**Reply by jordandsullivan** [reply]
Done: https://github.com/unitaryfund/mitiq/pull/2301

**Comment by purva-thakre** [comment]
Closing this as the discussion has been resolved.
